biogeochem/EDPhysiologyMod.F90

```diff
@@ -2247,43 +2247,56 @@ subroutine fragmentation_scaler( currentPatch, bc_in)
     !
     ! !LOCAL VARIABLES:
     logical  :: use_century_tfunc = .false.
+    logical  :: use_hlm_soil_scalar = .false. ! Use hlm input decomp fraction scalars
     integer  :: j
-    integer  :: ifp                   ! Index of a FATES Patch "ifp"
-    real(r8) :: t_scalar
-    real(r8) :: w_scalar
-    real(r8) :: catanf                ! hyperbolic temperature function from CENTURY
-    real(r8) :: catanf_30             ! hyperbolic temperature function from CENTURY
-    real(r8) :: t1                    ! temperature argument
+    integer  :: ifp                          ! Index of a FATES Patch "ifp"
```

Comment:
```suggestion
    integer  :: ifp                          ! Index of a FATES Patch "ifp"
    
    integer :: soil_layer_index     ! Index of soil layer for w_scaler 
```
```suggestion
    integer  :: ifp                          ! Index of a FATES Patch "ifp"
```

Reply:
Good idea.  Yup, this should go in CWDOut.  I'll add it in and push an update.